internal/types/metadata.go

```diff
@@ -33,7 +33,7 @@ type AppMetadata struct {
 	Version     string
 	Name        string
 	Description string
-	Namespace   string `yaml:"namespace"`
```

Comment:
Was this change intended? In the same commit?

Reply:
Yeah right it's in the same commit... It's just something I spotted. But there's a comment on it in the commit message 😄